package.json

```diff
@@ -10,6 +10,11 @@
     "base64",
     "gzip"
   ],
+  "dependencies": {
+    "stream":"~0.0.2",
+    "util": "~0.10.3",
+    "string_decoder": "~0.10.31"
```

Comment:
No, including them explicitly does nothing in Node (unless you're messing with the require system somehow). The Node.js module resolution algorithm will always use a native module if the name exists: https://nodejs.org/dist/latest-v6.x/docs/api/modules.html#modules_core_modules – packages that have the same name as core modules are dangerous anomalies.